src/app/item-popup/DeepSightHarmonizerIcon.m.scss

```diff
@@ -0,0 +1,14 @@
+@use '../variables.scss' as *;
+
+.deepsightHarmonizerIcon {
+  flex-grow: 1;
+  max-width: fit-content;
+
+  img {
+    display: grid;
```

Comment:
Hmm, why have the image display as a grid? It won't have children to lay out.

Reply:
My bad, this was an alignment hack when playing with different layouts I forgot to correct